concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -913,56 +928,90 @@ jobs:
                     < ${state}-tfstate/${state}.tfstate \
                     > terraform-outputs/${state}.tfvars.sh
                 done
-      - task: generate-manifest
-        config:
-          platform: linux
-          image: docker:///governmentpaas/spruce
-          inputs:
-            - name: paas-cf
-            - name: terraform-outputs
-            - name: cf-secrets
-            - name: cf-certs
-            - name: bosh-CA
-          outputs:
-            - name: cf-manifest
-          params:
-            MANIFEST_STUBS: |
-              ./terraform-outputs/*.yml
-              ./cf-secrets/cf-secrets.yml
-              ./certs/*.yml
-              ./paas-cf/manifests/cf-manifest/deployments/env-specific/{{cf_env_specific_manifest}}
-          run:
-            path: sh
-            args:
-              - -e
-              - -c
-              - |
-                mkdir certs
-                echo "Extracting extant certs"
-                tar -xvzf bosh-CA/bosh-CA.tar.gz -C certs
-                tar -xvzf cf-certs/cf-certs.tar.gz -C certs
 
-                cd certs
-                for file in *.crt; do
-                  cn=${file%.crt}
-                  yaml_key_name=$(echo ${cn} | tr A-Z- a-z_)
+      - aggregate:
+        - task: generate-cloud-config
+          config:
+            platform: linux
+            image: docker:///governmentpaas/spruce
+            inputs:
+              - name: paas-cf
+              - name: terraform-outputs
+              - name: cf-secrets
+            outputs:
+              - name: cloud-config
+            params:
+              MANIFEST_STUBS: |
+                ./paas-cf/manifests/cf-manifest/cloud-config/*.yml
+                ./terraform-outputs/*.yml
+                ./cf-secrets/cf-secrets.yml
+            run:
+              path: sh
+              args:
+                - -e
+                - -c
+                - |
+                  echo "Generating cloud config..."
 
-                  echo "Converting ${cn} certs to YAML"
-                  ../paas-cf/concourse/scripts/file-to-yaml.sh secrets ${yaml_key_name}_key ${cn}.key > ${cn}_key.yml
-                  ../paas-cf/concourse/scripts/file-to-yaml.sh secrets ${yaml_key_name}_cert ${cn}.crt > ${cn}_cert.yml
-                done
+                  # FIXME: Remove eval after https://github.com/concourse/concourse/issues/360
```

Comment:
In order to get rid of the quotes added by concourse, we need to do the additional eval...